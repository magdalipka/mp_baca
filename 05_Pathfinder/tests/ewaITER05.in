5 8
1 0 1 0 1
1 1 0 0 0
1 1 0 1 0
0 0 0 1 1
1 1 0 1 1
1 0 0 0 1
0 0 1 0 0
0 1 1 1 0
1
i 2 4 4 5

