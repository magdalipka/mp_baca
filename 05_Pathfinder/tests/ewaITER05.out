i N N E E S
